### DISCUSSION TITLE
Tasmota Enabled Home

### DISCUSSION THREAD

**Discussion by NsinghP** [arendst/Tasmota#19392] [Show and tell]
Time to share how I have used Tasmota in my home with the help of great engineers of Tasmota team! Special Thanks to @arendst  @barbudor , @sfromis , @Jason2866 , @ascillato , @benzino77 and other Tasmota users who have taken the time to answer my questions, sometimes very silly questions that I have posted in this forum.

I discovered Tasmota sometime in 2020 and started experimenting with it soon thereafter, I had zero knowledge about soldering and home automation at this point. However, playing around with ESP8266 and Tasmota was so easy that soon I had a good understanding of how to flash and use ESP8266 with Tasmota. 
 
Currently, I am using Eleven (11) ESP8266 based devices to control 24 appliances in my home. This includes, Lights, Air conditioners, Ceiling fans and Water Heaters. Eleven Esp8266 Devices include Five (5) Custom devices*, where I ordered a custom PCB and painstakingly soldered ESP8266 chip and then used online Tasmocompiler to make custom .bin file and used Tasmotizer to flash these devices. Another Five (5) devices are modified, commercially available Tinxy 30Amp devices that I have flashed with custom .bin Tasmota. Remaining one (1) Device is a Wipro RGB 9Watt bulb that I have used Tuyaconvert to flash with stock Tasmota.  I plan to add 2 More Tasmota Devices to this setup in future to control Electric water heaters.

*One of the custom device controls 7 appliances (2 Ceiling fans, 4 wall lights, and 1 ceiling light) with 5 Physical wall Switches and 2 soft switches!). This one is against official advice!, you can read all about this one here [https://github.com/arendst/Tasmota/discussions/12473](https://github.com/arendst/Tasmota/discussions/12473) .

I am using either Home Assistant with MQTT Server, or Home Remote App to control the appliances locally and Alexa to control the devices from anywhere on the internet. Home Assistant uses MQTT server, Home Remote App and Alexa uses Home Wifi/IP network with “Hue Emulation” feature enabled on all Tasmota devices. 

I have also installed an old IPad 2nd generation on the wall and installed Home Remote App on it, in Guided access mode. It has made life very easy for non-technical users in the house.  
(Note: Home Remote App is the only app, that I know of, which is still downloadable from Appstore and still works on older Ipads.)

I can use voice commands with Alexa to control all these Tasmota devices, other than that, here are some pictures of different methods I can use to control my devices:

(1) Physical on/off 2-way wall switch:
![wall switches](https://github.com/arendst/Tasmota/assets/69470381/18e61e1f-4ce4-4f71-a7ba-09dcd998dfa1)

(2) Wall mounted Touch Switch
![Touch Switches on wall](https://github.com/arendst/Tasmota/assets/69470381/5efb297c-ec4b-4e52-8ada-c89072a9e054)

(3)Tasmota Device Manager on Win11 PC, 
![Tasmota Device Manager 1](https://github.com/arendst/Tasmota/assets/69470381/61c6c636-a3b9-4891-af7a-1d875c03351b)

(4) MQTT Explorer On Win11PC/Iphone/Android Phone
![MQTT Explorer](https://github.com/arendst/Tasmota/assets/69470381/c77ada72-b507-4370-8caa-01429365a466)

(5) Home Remote App on Ipad/Iphone or Android Tab/Phone
![Ipad wall mounted closeup](https://github.com/arendst/Tasmota/assets/69470381/1d811219-b113-4416-8386-c0e9c39ec82a)

(6) Home Assistant on Raspberry Pi
![Home Asistant Overview](https://github.com/arendst/Tasmota/assets/69470381/c3146295-0811-42c2-8307-c4ebefaa78e7)

(7) Alexa on Windows 11 PC/Iphone/Android phone
![Alexa on PC](https://github.com/arendst/Tasmota/assets/69470381/5cd83d32-38c1-4c66-8808-9264b6c941ac)

(8) Wall mounted very old Ipad:
![ipad on wall](https://github.com/arendst/Tasmota/assets/69470381/85962278-369e-4130-9e7e-3850171242d1)

**Comment by barbudor** [comment]
Good job
